src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Select.test.js

```diff
@@ -68,6 +68,22 @@ test('The component should render with an icon', () => {
     expect(document.body.innerHTML).toBe('');
 });
 
+test('The component should render no option text when no option is available', () => {
+    const isOptionSelected = jest.fn().mockReturnValue(false);
+    const onSelect = jest.fn();
+    const select = mount(
+        <Select
+            displayValue="My text"
+            icon="su-plus"
+            isOptionSelected={isOptionSelected}
+            onSelect={onSelect}
+        >
+        </Select>
+    );
+    expect(select.render()).toMatchSnapshot();
```

Comment:
should i remove it?

Reply:
I would say try to click the button and see what's then in the snapshot :slightly_smiling_face: The text you have defined should show up there somehow. Otherwise I would not do a snapshot test, and check using the `prop` function.